M2/Macaulay2/d/actors.d

```diff
@@ -1010,6 +1014,7 @@ installMethodFun2(arg1:Expr,args:CodeSequence):Expr := (
      else buildErrorPacket("expected left hand parameter to be a function, type, or a hash table"));
 installMethodFun(args:CodeSequence):Expr := installMethodFun2(eval(args.1),args);
 InstallMethodFun = installMethodFun;
+setup(ColonEqualS,InstallMethodFun);
```

Comment:
Was this really not a symbol before and is being added now?!

Reply:
try `pseudocode` on something like
```
() -> (
toList List := identity
)
```
(+disassembling before this PR).
Currently it doesn't have a symbol for `InstallMethodFun` so will display nothing. after this PR it will show the symbol `:=".